src/Modules/CrestApps.OrchardCore.AI.Chat/Views/AIChatSessionsList.cshtml

```diff
@@ -32,6 +32,12 @@
                     </div>
                     <div>
                         <a asp-action="Index" asp-controller="Admin" asp-route-profileId="@session.ProfileId" asp-route-sessionId="@session.SessionId" class="btn btn-sm btn-success">@T["View"]</a>
+                        <a asp-action="Delete"
```

Comment:
Also, this button should only be visible to authenticated users only. Anonymous users can't use this function.